modules/ROOT/pages/kubernetes/deploy-cluster.adoc

```diff
@@ -102,7 +102,7 @@ Regular node::
 --
 [source,console]
 ----
-kubectl create secret generic sgw-config --from-file sgw-config.json
+kubectl create secret generic sgw-config -f sgw-config.json
```

Comment:
This one is actually right! It creates a secret from a file, it's just the deployment ones that were wrong.

Reply:
Back when we were working on this, I thought `--from-file` would work on all commands and be preferable to `-f` as it is more explicit. But given it's not the case let's use `-f` everywhere for consistency.